examples/with-loading/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "with-loading",
+  "version": "0.0.0",
+  "description": "",
+  "scripts": {
+    "dev": "../../dist/bin/next"
```

Comment:
Okay that makes sense. I will right another script to play with current source code.